bin/opencfp

```diff
@@ -9,13 +9,19 @@ use OpenCFP\Environment;
 use OpenCFP\Console\Application;
 use Symfony\Component\Console\Input\ArgvInput;
 use OpenCFP\Application as ApplicationContainer;
+use OpenCFP\Console\Command;
+use OpenCFP\Domain\Services;
 
 $basePath = realpath(dirname(__DIR__));
 $input = new ArgvInput();
 $environment = $input->getParameterOption(array('--env'), getenv('CFP_ENV') ?: 'development');
 
 $container = new ApplicationContainer($basePath, Environment::fromString($environment));
 $app = new Application($container);
+
+$app->addCommands([
+    new Command\UserCreateCommand($container[Services\AccountManagement::class]),
```

Comment:
Adding commands here for now, later we could also just fetch the application from the container, if we wanted to.